src/Compilers/CSharp/Test/CommandLine/project.json

```diff
@@ -1,5 +1,7 @@
 ﻿{
   "dependencies": {
+    "xunit": "2.1.0",
```

Comment:
General question: what was the philosophy you chose for whether we should restate things that a dependent project already has? Like I'd guess in this case that some TestUtilities project already had xunit?